xdem/examples.py

```diff
@@ -0,0 +1,78 @@
+import asyncio
+import os
+
+import geopandas as gpd
+import matplotlib.pyplot as plt
+import rasterio as rio
+
+
+async def async_load_svalbard():
+    """Load the datasets asynchronously."""
+    # The URLs for the files to load
+    urls = {
+        "ref": "zip+https://publicdatasets.data.npolar.no/kartdata/S0_Terrengmodell/Mosaikk/NP_S0_DTM20.zip!NP_S0_DTM20/S0_DTM20.tif",
+        "tba": "zip+https://publicdatasets.data.npolar.no/kartdata/S0_Terrengmodell/Historisk/NP_S0_DTM20_199095_33.zip!NP_S0_DTM20_199095_33/S0_DTM20_199095_33.tif",
+        "outlines": "http://public.data.npolar.no/cryoclim/CryoClim_GAO_SJ_1990.zip"
+    }
+
+    # The output filepaths to save the files at.
+    output_paths = {
+        "ref": "examples/Longyearbyen/data/DEM_2009_ref.tif",
+        "tba": "examples/Longyearbyen/data/DEM_1995.tif",
+        "outlines": "examples/Longyearbyen/data/glacier_mask/CryoClim_GAO_SJ_1990.shp"
+    }
+    # The bounding coordinates to crop the datasets.
+    bounds = {
+        "west": 502810,
+        "east": 529450,
+        "south": 8654330,
+        "north": 8674030
+    }
+
+    async def crop_dem(input_path, output_path, bounds):
+        dem = rio.open(input_path)
+
+        upper, left = dem.index(bounds["west"], bounds["north"])
+        lower, right = dem.index(bounds["east"], bounds["south"])
```

Comment:
would this somehow work directly with the Raster.crop() method (and thus DEM.crop() method)? 
maybe it's too specific with the window reading

Reply:
I wanted to avoid loading the entire raster into memory. Would `Raster.crop()` read a window appropriately?

Reply:
I wanted to test this actually. I can have a look.